sdks/python/src/honcho/async_client/peer.py

```
@@ -0,0 +1,331 @@
+
+        Makes an async API call to search for messages in this peer's global representation.
+
+        Args:
+            query: The search query to use
+
+        Returns:
+            An AsyncPage of Message objects representing the search results.
+            Returns an empty page if no messages are found.
+        """
+        messages_page = await self._client.workspaces.peers.search(
+            self.id, workspace_id=self.workspace_id, query=query
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)